layouts/_default/baseof.css

```diff
@@ -303,12 +304,11 @@ a:hover,
 
 /* #region focus */
 
+:focus,
 .btn:focus,
 a:focus {
```

Comment:
Not sure we should create a global focus outline like this? What about `input` boxes for instance? They should probably not have this outline added.

Reply:
That's true, I removed this.